dotnet/src/webdriver/DriverService.cs

```diff
@@ -363,23 +367,29 @@ private void Stop()
                 {
                     Uri shutdownUrl = new Uri(this.ServiceUrl, "/shutdown");
                     DateTime timeout = DateTime.Now.Add(this.TerminationTimeout);
-                    while (this.IsRunning && DateTime.Now < timeout)
+                    using (var httpClient = new HttpClient())
```

Comment:
Reuse the instance on class level?

Reply:
Because we are using 2 different HttpClient instances to start & stop the service? Not sure it saves much, but we could.

Reply:
Let's keep it simple for now.